src/core/edgeNetwork/sendEdgeNetworkRequestFactory.js

```diff
@@ -94,7 +116,11 @@ export default ({
           // be thrown here which should ultimately reject the promise that
           // was returned to the customer for the command they executed.
           processWarningsAndErrors(response);
-          return response;
+
+          const first = processReturnValue(...segmentedReturnValues[0]);
+          const second = processReturnValue(...segmentedReturnValues[1]);
```

Comment:
Maybe call these `lifecycleOnResponseReturnValues` and `consumerOnResponseReturnValues` instead of `first` and `second`?

Reply:
yes, sure